.pipelines/int-e2e.yml

```diff
@@ -0,0 +1,42 @@
+# Azure DevOps Pipeline running e2e tests
```

Comment:
I think we should try to have 1 and unique pipeline for e2e and make it flexible either for PR/INT/PROD .... 

What are the differences between this one and `pr-e2e.yaml` ?

Reply:
done

Reply:
Lets make sure to re setup the PR E2E pipeline when this is merge to point to the new `e2e.yml` file